lsm-tree.cabal

```diff
@@ -57,6 +57,7 @@ library
     , io-classes            ^>=1.2
     , lsm-tree:bloomfilter
     , primitive             ^>=0.8
+    , unix                  ^>=2.8
```

Comment:
Doesn't work on Windows and MacOS -- maybe we should disable windows/macos in CI from now on, until we get to using `fs-sim` and `fs-api`